application/forms/Config/DeploymentConfigForm.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+// Icinga Web JIRA Integration | (c) 2023 Icinga GmbH | GPLv2
+
+namespace Icinga\Module\Jira\Forms\Config;
+
+use Icinga\Application\Config;
+use Icinga\Module\Jira\RestApi;
+use Icinga\Web\Notification;
+use ipl\Validator\CallbackValidator;
+use ipl\Web\Compat\CompatForm;
+
+class DeploymentConfigForm extends CompatForm
+{
+    /**
+     * @var string Deployment type of the Jira Software cloud: or server
+     */
+    protected $deploymentType;
+
+    /**
+     * @var int Whether create issue meta uses legacy method
+     */
+    protected $legacy;
+
+    public function __construct()
+    {
+        $config = Config::module('jira');
+        if ($config->hasSection('deployment')) {
+            $this->deploymentType = $config->get('deployment', 'type');
+            $this->legacy = (int) $config->getSection('deployment')->get('legacy');
+        } else {
+            $this->deploymentType = 'cloud';
+        }
+    }
+
+    protected function assemble()
+    {
+        $this->addElement('select', 'deployment_type', [
+            'class'         => 'autosubmit',
+            'label'         => t('Deployment Type'),
+            'description'   => t('Select deployment type'),
+            'multiOptions'  => ['cloud' => t('Cloud'), 'server' => t('Server')],
+            'required'      => true,
+            'value'         => $this->deploymentType,
+            'validators'    => [
+                'Callback' => function ($value, $validator) {
+                    /** @var CallbackValidator $validator */
+                    $serverInfo = RestApi::fromConfig()->get('serverInfo')->getResult();
+
+                    if ($value !== strtolower($serverInfo->deploymentType)) {
+                        $validator->addMessage(
+                            t('Jira Software seems to be deployed differently from the deployment type chosen')
+                        );
+                        return false;
+                    }
+
+                    return true;
+                }
+            ]
+        ]);
+
+        if ($this->getValue('deployment_type') === 'cloud') {
+            $this->addElement('checkbox', 'legacy', [
+                'label'         => t('Legacy compatibility'),
+                'description'   => t('Enable/disable legacy compatibility'),
```

Comment:
Honestly, in which way improves `Enable/disable` the description? This just states the obvious, since it's a checkbox element. And it doesn't explain why the *legacy compatibility* might be needed. Try to use a description that is good enough that anyone seeing this for the first time knows what to do without taking a look at the (non-existing) documentation.